pyart/aux_io/rainbow_wrl.py

```diff
@@ -0,0 +1,493 @@
+    file version.
+
+    Data types read by this routine:
+    Reflectivity: dBZ, dBuZ, dBZv, dBuZv
+    Velocity: V, Vu, Vv, Vvu
+    Spectrum width: W, Wu, Wv, Wvu
+    Differential reflectivity: ZDR, ZDRu
+    Co-polar correlation coefficient: RhoHV, RhoHVu
+    Co-polar differential phase: PhiDP, uPhiDP, uPhiDPu
+    Specific differential phase: KDP, uKDP, uKDPu
+    Signal quality parameters: SQI, SQIu, SQIv, SQIvu
+    Temperature: TEMP
+    Position of the range bin respect to the ISO0: ISO0
+
+    Parameters
+    ----------
+    filename : str
+        Name of the RAINBOW file to read.
+    field_names : dict, optional
+        Dictionary mapping RAINBOW field names to radar field names. If a
+        data type found in the file does not appear in this dictionary or has
+        a value of None it will not be placed in the radar.fields dictionary.
+        A value of None, the default, will use the mapping defined in the
+        Py-ART configuration file.
+    additional_metadata : dict of dicts, optional
+        Dictionary of dictionaries to retrieve metadata during this read.
+        This metadata is not used during any successive file reads unless
+        explicitly included.  A value of None, the default, will not
+        introduct any addition metadata and the file specific or default
+        metadata as specified by the Py-ART configuration file will be used.
+    file_field_names : bool, optional
+        True to use the MDV data type names for the field names. If this
+        case the field_names parameter is ignored. The field dictionary will
+        likely only have a 'data' key, unless the fields are defined in
+        `additional_metadata`.
+    exclude_fields : list or None, optional
+        List of fields to exclude from the radar object. This is applied
+        after the `file_field_names` and `field_names` parameters.
+
+
+    Returns
+    -------
+    radar : Radar
+        Radar object containing data from RAINBOW file.
+
+    """
+
+    # check that wradlib is available
+    if not _WRADLIB_AVAILABLE:
+        raise MissingOptionalDependency(
+            "wradlib is required to use read_rainbow_wrl but is not installed")
+
+    # test for non empty kwargs
+    _test_arguments(kwargs)
+
+    # check if it is the right file. Open it and read it
+    bfile = os.path.basename(filename)
+    supported_file = (bfile.endswith('.vol') or bfile.endswith('.azi')
+                      or bfile.endswith('.ele'))
+    if not supported_file:
+        raise ValueError(
+            'Only data files with extension .vol, .azi or .ele are supported')
+
+    # create metadata retrieval object
+    if field_names is None:
+        field_names = RAINBOW_FIELD_NAMES
+    filemetadata = FileMetadata('RAINBOW', field_names, additional_metadata,
+                                file_field_names, exclude_fields)
+
+    rbf = wrl.io.read_Rainbow(filename, loaddata=True)
```

Comment:
> Rainbow raw data files tipically contain only one moment and sooner or later you have to read it so I do not think there would be a major improvement.

Yes, didn't thought about having the moments in different files. We can try anyway and have some performance checks.

> If we created a function that would read the files of several moments at once to load the Radar object then probably would be worth it.

This is even more intriguing idea. We should definitely try this out. @jfigui I have some multi-scan .vol files as well as single scan .azi and .ele files to test. How is your data filename build up? Here I have eg. `YYYYMMDDHHMMSS00mom.vol`. Then we could build in some magic to retrieve all the other corresponding files. Or we can try to give a filelist to the reading function. Many opportunities. I'll give it a try, but this might take some time.